setup.cfg

```diff
@@ -50,15 +50,17 @@ datalad = datalad
 doc =
     nbsphinx
     packaging
-    pydot >=1.2.3
+    pydot >= 1.2.3
     pydotplus
-    sphinx >=1.5.3
+    sphinx >= 1.5.3, < 3
     sphinx-argparse
     sphinx_rtd_theme
     sphinxcontrib-napoleon
 docs =
     %(doc)s
-duecredit = duecredit
+duecredit =
+    duecredit
+    citeproc-py != 0.5.0
```

Comment:
is this implemented?

Reply:
I think this issue has been fixed?

Reply:
i wasn't sure after following https://github.com/brechtm/citeproc-py/issues/94

Reply:
Right, I guess it will fail for citeproc-py 0.5.0, but do we generally add carve-outs like this for things up the dependency tree that have bugs?

Reply:
sure - we can probably just remove this hack now that 0.5.1 is out.